ginga/Control.py

```diff
@@ -850,16 +866,22 @@ def add_image(self, imname, image, chname=None, silent=False):
         if chname is None:
             channel = self.get_channelInfo()
             if channel is None:
-                raise ValueError("Need to provide a channel name to add the image")
+                raise ValueError("Need to provide a channel name to add "
+                                 "the image")
             chname = channel.name
 
+        # Initialize MODIFIED header keyword for image-modified callback.
+        # This is for consistent keyword display in Contents, and used in
+        # 'image-modified' callback.
+        image.metadata['header']['MODIFIED'] = None
```

Comment:
If I make it too specific to FITS specific calls, then it might break if JPEG emits the same callback, no? That is the reason I didn't use `set_keyword()`. Good points on the rest, I'll have to think about it more.